CHANGES.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+# 0.12.4
+
+### New
+
+- [helm] The compute log manager now defaults to a `NoOpComputeLogManager`. It did not make sense to default to the `LocalComputeLogManager` as pipeline runs are executed in ephemeral jobs, so logs could not be retrieved once these jobs were cleaned up. To have compute logs in a Kubernetes environment, users should configure a compute log manager that uses a cloud provider. 
```

Comment:
@rexledesma would it be more accurate to list this as a breaking change?

Reply:
I wouldn't classify it as a breaking change since this is the current behavior in Kubernetes - this change just makes it more explicit.

Reply:
I don't think so - the expectation is there is no perceived change in behavior to users 

Maybe "bugfix" instead of "new" tho

Reply:
Spoke offline - we're comfortable highlighting this as "New"